runhouse/docker/slim/public-key-auth/Dockerfile

```diff
@@ -51,7 +50,10 @@ RUN echo "[supervisord]" > /etc/supervisor/conf.d/supervisord.conf && \
     echo "[program:sshd]" >> /etc/supervisor/conf.d/supervisord.conf && \
     echo "command=/usr/sbin/sshd -D" >> /etc/supervisor/conf.d/supervisord.conf && \
     echo "stdout_logfile=/var/log/sshd.log" >> /etc/supervisor/conf.d/supervisord.conf && \
-    echo "stderr_logfile=/var/log/sshd.err" >> /etc/supervisor/conf.d/supervisord.conf
+    echo "stderr_logfile=/var/log/sshd.err" >> /etc/supervisor/conf.d/supervisord.conf  && \
+    echo "[program:nginx]" >> /etc/supervisor/conf.d/supervisord.conf && \
```

Comment:
Should starting nginx be guarded by an build-time ARG, similar to RUNHOUSE_VERSION?